### PR TITLE
[mime-db] Add exported object

### PR DESCRIPTION
**Pull request by LinusU**
Replaces #42868, and uses `ReadonlyArray<...>` instead of `readonly ...[]` for maximum compatibility.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/jshttp/mime-db#usage https://github.com/jshttp/mime-db#data-structure
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

File: types/mimos/index.d.ts [hunk 1]
```diff
@@ -1,10 +1,11 @@
 // Type definitions for mimos 3.0
 // Project: https://github.com/hapijs/mimos
 // Definitions by: AJP <https://github.com/AJamesPhillips>
+//                 Linus Unnebäck <https://github.com/LinusU>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.2
 
-import { DataStructure as MimeDbDataStructure } from 'mime-db';
+import database = require('mime-db');
```
Comment (SimonSchick) [inline types/mimos/index.d.ts]: This line shouldn't need to be changed, `import` still works with declaration merged exports.

File: types/hapi__mimos/index.d.ts [hunk 2]
```diff
@@ -2,10 +2,11 @@
 // Project: https://github.com/hapijs/mimos
 // Definitions by: AJP <https://github.com/AJamesPhillips>
 //                 Silas Rech <https://github.com/lenovouser>
+//                 Linus Unnebäck <https://github.com/LinusU>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.2
 
-import { DataStructure as MimeDbDataStructure } from 'mime-db';
+import database = require('mime-db');
```
Comment (SimonSchick) [inline types/hapi__mimos/index.d.ts]: This line shouldn't need to be changed, `import` still works with declaration merged exports.

**Review by SimonSchick — APPROVED** [review]
Much better now.
Minor comments remain.

**Comment by LinusU** [comment]
@SimonSchick  Imported using the module syntax ✔️

**Comment by LinusU** [comment]
ping @sandersn, @orta, @ElizabethSamuel-MSFT any idea why this haven't been merged yet? ❤️

**Comment by LinusU** [comment]
ping @AJamesPhillips @lenovouser ☺️

**Comment by AJamesPhillips** [comment]
@LinusU sorry I have no capacity to review this for the foreseeable future.

**Review by lenovouser — APPROVED** [review]
@LinusU sorry, have been sick lately. Looks very good!

**Comment by LinusU** [comment]
No worries, thanks!

**Pull request merged**